### DISCUSSION TITLE
How to change menu bar color?

### DISCUSSION THREAD

**Discussion by RottenYogurt** [sumatrapdfreader/sumatrapdf#2441] [Q&A]
Last time I success to change the background of Sumatra for pdf, epub. But I still want bright color from menu bar at the top of the window to be darker but cannot found in sumatraPDF-settings.txt file. Does sumatra have  any function to do this?
(The region I want to change is shaded with red pencil)
![화면 캡처 2022-03-28 233552](https://user-images.githubusercontent.com/99065262/160422334-b7df1a15-c75b-491f-af1c-7b7d98f99a21.png)

**Comment by GitHubRulesOK** [comment]
those areas are generally controlled by windows however when system colors is used in newer windows it usually results in a mess however is now improving in windows 11
![image](https://user-images.githubusercontent.com/12754913/161154366-5c06864d-901b-4c7d-817e-066efddf1ab5.png)
 so the default is false see https://github.com/sumatrapdfreader/sumatrapdf/issues/2241#issuecomment-962503365 the topic was covered hundreds of times and open issue is https://github.com/sumatrapdfreader/sumatrapdf/issues/743 but old forum is now impossible to search so posting one custom example of using windows blinds here https://forum.sumatrapdfreader.org/t/change-color-of-slider-on-scroll-bar/4232
![image](https://user-images.githubusercontent.com/12754913/160429486-fef53fd4-f754-4626-a443-beee050df275.png)

**Reply by RottenYogurt** [reply]
Does window blinds mean a third party software to customize window menu bar?

**Reply by GitHubRulesOK** [reply]
there are several good apps for theming from stardock apart from docking bars https://www.stardock.com/products/windowblinds/ and https://www.stardock.com/products/curtains/

**Reply by RottenYogurt** [reply]
Thank you so much !!

**Comment by GitHubRulesOK** [comment]
Just to make the point about windows 11 here I am using bblean which is an old window7 desktop but plays nicely with SumatraPDF two windows same time on one screen
![image](https://user-images.githubusercontent.com/12754913/161159417-bbb07898-2a50-4696-be80-5ea25d763771.png)
